Lib/test/test_pty.py

```diff
@@ -139,6 +205,13 @@ def test_basic(self):
         # to ignore this signal.
         os.close(master_fd)
 
+        if winsz:
+            winsz = struct.pack("HHHH", current_stdin_winsz.lines,
+                                current_stdin_winsz.columns, 0, 0)
+            fcntl.ioctl(pty.STDIN_FILENO, TIOCSWINSZ, winsz)
+
+        # pty.openpty() passed.
```

Comment:
I don't think this comment adds any additional context for the reader, and can be removed.

Reply:
Gotcha. I did this because the test_fork() function also has a similar comment at the end thereof. Should I remove that one as well?